hw_diag/tasks.py

```diff
@@ -92,4 +93,6 @@ def perform_hw_diagnostics():
     with open('diagnostic_data.json', 'w') as f:
         json.dump(diagnostics, f)
 
+    upload_diagnostics(diagnostics)
```

Comment:
What's the frequency of this?

Reply:
Depends on how often diagnostics is ran in APScheduler, currently every 5 mins.